### PR TITLE
Va 49 scroll fix

### PR DESCRIPTION
**Pull request by nibivi77**
## Description

Fixed the scroll to bottom of chat messages.

## Notes

- There was a conflict with the animation making it look a little weird so I commented it out for the time being.

### CODE REVIEW

**Review by neilmispelaar — APPROVED** [review]
Looks great!

Great work! Thank you for this. Sorry about the annimations - we'll put them back in on Monday for sure when we figure out the jerky annimation thing.

File: src/components/organisms/ConversationWindow.vue [hunk 1]
```diff
@@ -152,6 +152,16 @@ export default {
       tabbable.value = -1;
     }
 
+    function scrollToBottom() {
+      const list = document.querySelector("#conversationWindow");
```
Comment (TishamIslam) [inline src/components/organisms/ConversationWindow.vue:156]: should use `document.getElementById("conversationWindow")`

Comment (neilmispelaar) [reply src/components/organisms/ConversationWindow.vue:156]: Great catch! Would we have more than one dynamic convo window in the dom? If there were two chat messages per say

Comment (neilmispelaar) [reply src/components/organisms/ConversationWindow.vue:156]: SOrry - chat windows - not chat messages

Comment (TishamIslam) [reply src/components/organisms/ConversationWindow.vue:156]: We use v-if for switching the chat window/message lists, so we'll always have at most one element with the conversationWindow id or the messageList id. Related to [this PR](https://github.com/Toad-Labs/vc-frontend-static/pull/153) where I used the fact that there's either one of those ids to focus on either window 
`const contentWindow = document.getElementById("conversationWindow") ||
          document.getElementById("messageList");`

Comment (neilmispelaar) [reply src/components/organisms/ConversationWindow.vue]: Thanks for the clarification

Benoit - I agree with Tisham that the document.getElementById is the better way to go

Comment (nibivi77) [reply src/components/organisms/ConversationWindow.vue]: How about what I just pushed? I am using the already declared ref.

Comment (neilmispelaar) [reply src/components/organisms/ConversationWindow.vue]: Oh lol this has already been addressed in the latest commit

Comment (TishamIslam) [reply src/components/organisms/ConversationWindow.vue]: Yup, that works better lol, good catch!

Comment (neilmispelaar) [reply src/components/organisms/ConversationWindow.vue]: I want to hit the resolve conversation button but I think Tisham or you should do the honours

**Pull request merged**